src/supermarket/config/initializers/content_security_policy.rb

```diff
@@ -8,7 +8,10 @@
   policy.default_src :self, :https
   policy.font_src :self, :https, :data, "http://fonts.gstatic.com"
   policy.img_src :self, :https, :data, "www.googletagmanager.com"
-  policy.script_src :self, :https, "https://www.googletagmanager.com", "https://www.google-analytics.com", "http://cdn.segment.com"
+  policy.connect_src :self, :https, "wss://ws.hotjar.com"
+  policy.script_src :self, :blob, :https, :unsafe_eval, "https://www.googletagmanager.com",
```

Comment:
`unsafe_eval` is very risky, add all the JS manually.

Reply:
We won't be able to add the js link manually as the external script is  being injected by gtm and we don't have control over it.  As the nonce approaches are also not feasible here, we don't have a choice other than `unsafe_eval`
https://developers.google.com/tag-platform/security/guides/csp

<img width="737" alt="Screenshot 2023-12-28 at 12 14 57 PM" src="https://github.com/chef/supermarket/assets/96620338/d02b30b3-3e66-411f-a949-4e84866f3f19">

Reply:
@saghoshprogress @RoyShravani 

I have removed the unsafe_eval from here, as Megan tested chef-manage on acceptance env and confirmed that even with unsafe_eval errors in the issues tab of browser console, the GTM functionalities are working correctly.

We do not need unsafe_eval here.